api/src/main/java/bio/terra/tanagra/service/underlay/Underlay.java

```diff
@@ -43,17 +45,17 @@
    */
   public abstract ImmutableMap<Relationship, ForeignKey> foreignKeys();
 
+  /** Map from entities to the filters schema that supports the entity. */
```

Comment:
nit: I would add ", if any", assuming that it is legal for an entity to omit a filter schema

Reply:
good point, added.